gen2-human-pose/main.py

```diff
@@ -167,6 +169,12 @@ def get_frame():
 
             if cv2.waitKey(1) == ord('q'):
                 break
+            elif cv2.waitKey(2) == ord('t'):
```

Comment:
`cv2.waitKey(X) ` [waits](https://docs.opencv.org/2.4/modules/highgui/doc/user_interface.html#waitkey) for X milliseconds for keypress. 
Change to: 
```
key = cv2.waitKey(1)
if key == ord('q'):
     break
elif key == ord('t'):
```

Reply:
And thanks for the change. Otherwise LGTM

Reply:
Thanks, changed the file